packages/wordpress-external-dependencies-plugin/index.js

```diff
@@ -105,6 +104,22 @@ class WordPressExternalDependenciesPlugin {
 	}
 }
 
+/**
+ * Given a string, returns a new string with dash separators converted to
+ * camelCase equivalent. This is not as aggressive as `_.camelCase` in
+ * converting to uppercase, where Lodash will also capitalize letters
+ * following numbers.
+ *
+ * From @wordpress/scripts@3.1.0
```

Comment:
Maybe add a comment why we're not using that dep instead?

Reply:
I expect this plugin to be replaced by a core-provided one soon, but we can add that 👍 

```suggestion
 * From @wordpress/scripts@3.1.0
 * Duplicated here to avoid depending on @wordpress/script with bloated dependencies.
```